Example/PaymentSheet Example/PaymentSheet Example-Shard1.xctestplan

```diff
@@ -31,6 +31,8 @@
     {
       "parallelizable" : true,
       "skippedTests" : [
+        "EmbeddedUITests",
+        "EmbeddedUITests\/testUpdate()",
```

Comment:
Hmm, this doesn't look right. Why is it including the test name `testUpdate` not just the class?

Reply:
IDK, this is what Xcode did. In Xcode it looks the same as other tests:

![CleanShot 2024-10-25 at 17 47 30](https://github.com/user-attachments/assets/1433c644-89f9-40e9-9db4-673a3cae5c86)

Reply:
Adding another test to EmbeddedUITest automatically does the expected thing, too - skipped in shards 1, 2, 3 and added to shard 4.